### PR TITLE
Updating Hero Homepage 

### PR DESCRIPTION
**Pull request by codiejay**
# Updating Hero Homepage

show real data (image, etc)
remove dummy data
link user to profile
get ens image from user

closes #182, closes #182, closes https://github.com/EveripediaNetwork/issues/issues/182

### CODE REVIEW

File: src/components/Landing/Hero.tsx [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
 import { Wiki } from '@/types/Wiki'
 import shortenAccount from '@/utils/shortenAccount'
 import NextLink from 'next/link'
+import { useEnsAvatar } from 'wagmi'
+import { WikiImage } from '../WikiImage'
 
 const HeroCard = ({ wiki }: HeroProps) => {
-  const { image, avatarImage } = featuredArticle
+  const [{ data: avatar }] = useEnsAvatar({
+    addressOrName: '0x9fEAB70f3c4a944B97b7565BAc4991dF5B7A69ff',
```
Comment (kesar) [inline src/components/Landing/Hero.tsx:12]: please make sure you review your PR before tagging others to review.

there is a hardcoded address there

File: src/components/Landing/Hero.tsx [hunk 2]
```diff
@@ -23,17 +26,21 @@ const HeroCard = ({ wiki }: HeroProps) => {
...
-          bg="gray.100"
-          loading="lazy"
-          roundedTop="lg"
-          h={{ base: 80, lg: 'sm' }}
-        />
+        {wiki && (
+          <WikiImage
+            cursor="pointer"
+            flexShrink={0}
+            wiki={wiki}
```
Comment (kesar) [inline src/components/Landing/Hero.tsx:33]: this has changed in main, make sure to fix it

File: src/components/Landing/Hero.tsx [hunk 3]
```diff
@@ -23,17 +26,21 @@ const HeroCard = ({ wiki }: HeroProps) => {
...
+            h={{ base: '100%', lg: 400 }}
+            w={{ base: '100%', lg: '100%' }}
+            borderRadius="none"
+            overflow="hidden"
+          />
+        )}
         <Flex p="3" align="center" gap={4}>
-          <Avatar boxSize={10} src={avatarImage} />
+          <NextLink href={`/account/${wiki?.user?.id}`} passHref>
+            <Avatar boxSize={10} src={avatar || undefined} />
```
Comment (kesar) [inline src/components/Landing/Hero.tsx:42]: doesnt avatar just work?

Comment (codiejay) [reply src/components/Landing/Hero.tsx:42]: The type for the source attr is expecting a `string | undefined`. If we use just `Avatar` we get a type error.

**Comment by kesar** [comment]
image

site looks broken

**Comment by codiejay** [comment]
> image
> 
> site looks broken

image
This is what I have after merging with `main`. Also I think the image is this way cause the image hash attached to this promoted wiki is actually broken.

**Comment by kesar** [comment]
> > image
> > site looks broken
> 
> image
> 
> This is what I have after merging with `main`. Also I think the image is this way cause the image hash attached to this promoted wiki is actually broken.

that might be bcs u are logged in

File: src/components/Landing/Hero.tsx [hunk 4]
```diff
@@ -23,17 +26,19 @@ const HeroCard = ({ wiki }: HeroProps) => {
...
+          image={wiki?.images?.[0]?.id}
+          h={{ base: '100%', lg: 400 }}
+          w={{ base: '100%', lg: '100%' }}
+          borderRadius="none"
+          overflow="hidden"
         />
         <Flex p="3" align="center" gap={4}>
-          <Avatar boxSize={10} src={avatarImage} />
+          <NextLink href={`/account/${wiki?.user?.id}`} passHref>
+            <Avatar size="md" boxSize={10} src={avatar || undefined} />
```
Comment (kesar) [inline src/components/Landing/Hero.tsx:40]: why change the size? it wasnt in the specs

**Comment by kesar** [comment]
its broken on tablet and mobile version

**Comment by codiejay** [comment]
> image
> 
> vs
> 
> image

This is because the images to this particular wiki is broken at the moment. A potential fix is if we could edit that wiki and re-upload an image

**Comment by codiejay** [comment]
> > This is because the images to this particular wiki is broken at the moment. A potential fix is if we could edit that wiki and re-upload an image
> 
> not true
> 
> image

Screenshot 2022-04-04 at 16 25 37
Oh now it seems the promoted wiki has been updated because now if we check the image I attached to this reply the wiki we had before with the broken image has now changed.

**Review by kesar — APPROVED** [review]

**Pull request merged**